hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/security/acl/OzoneAccessAuthorizer.java

```diff
@@ -23,6 +23,13 @@
  * */
 public class OzoneAccessAuthorizer implements IAccessAuthorizer {
 
+  private static final OzoneAccessAuthorizer INSTANCE =
```

Comment:
Is it worth adding a comment here explaining why the constructor isn't private?

Reply:
Good point, but I think hardly anyone ever touches this class.